api/src/controllers/utils.ts

```diff
@@ -124,13 +123,35 @@ router.post(
 		const busboy = Busboy({ headers });
 
 		busboy.on('file', async (_fieldname, fileStream, { mimeType }) => {
-			try {
-				await service.import(req.params['collection']!, mimeType, fileStream);
-			} catch (err: any) {
-				return next(err);
-			}
-
-			return res.status(200).end();
+			const { path: filePath, cleanup } = await createTmpFile();
+
+			fileStream.pipe(fs.createWriteStream(filePath));
+
+			fileStream.on('end', async () => {
+				const workerPath = new URL('../utils/import-worker', import.meta.url);
```

Comment:
Here I didn't use `.js` extension since it does not work in development mode.
Without the file extension it works both while in development but also after the generation of distributed files.

@paescuj do you have any suggestion here or is it ok to have it like this?
Also, @nickrum if you have sometime can you also look at this? Asking since you are the ESM man 😅

Reply:
This is fine, for `Worker` the URL extension will be added if necessary 👍 
We'll have to take another look at this when it comes to bundling of the api, but let's not consider this a blocker for now.